starlite/signature.py

```diff
@@ -41,7 +41,7 @@
 
 UNDEFINED_SENTINELS = {Undefined, Signature.empty}
 SKIP_NAMES = {"self", "cls"}
-SKIP_VALIDATION_NAMES = {"request", "socket", "state", "scope"}
+SKIP_VALIDATION_NAMES = {"request", "socket", "state", "scope", "receive", "send"}
```

Comment:
This isn't necessary for the PR, but curious on thoughts on whether we should be validating these?

Reply:
how would we?